voluntera.Log

```diff
@@ -0,0 +1 @@
+
```

Comment:
You don't need to push the log file to GitHub because then every team member's local logs will start conflicting. Remove this file and push again and add it to gitignore file so that it doesn't get pushed other times.